wolfssl/wolfcrypt/settings.h

```diff
@@ -2671,6 +2671,16 @@ extern void uITRON4_free(void *p) ;
     #define NO_RC4
 #endif
 
+#if !defined(WOLFSSL_NO_ASYNC_IO) || defined(WOLFSSL_ASYNC_CRYPT) || \
```

Comment:
Consider adding `WOLFSSL_NO_ASYNC_IO` to documentation... and to the configure `--enable-lowresource`.

Reply:
Added to our macro docs and added to the low resource build.